src/test/java/net/minecraftforge/debug/world/RaidEventTest.java

```diff
@@ -0,0 +1,96 @@
+package net.minecraftforge.debug.world;
+
+import net.minecraft.Util;
+import net.minecraft.core.BlockPos;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.effect.MobEffectInstance;
+import net.minecraft.world.effect.MobEffects;
+import net.minecraft.world.entity.raid.Raid;
+import net.minecraft.world.entity.raid.Raider;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.world.RaidEvent;
+import net.minecraftforge.fml.common.Mod;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Set;
+
+@Mod("raid_event_test")
+public class RaidEventTest
```

Comment:
If you look in other test mods there is ususally a ``private static final boolean ENABLE = false;`` this is so that they can be disabled more easily.